### DISCUSSION TITLE
How to name elements inside nested blocks?

### DISCUSSION THREAD

**Discussion by aaty2020** [getbem/getbem.github.io#317] [Q&A]
How do you name the elements inside your nested blocks? Should you put the name of the parent block class as a prefix or the name of a higher block class? For example, in the code below, how should you name the element p (block1__elemanname or block2__elementname )?
```
<div class="block1">
  <div class="block2">
    <p class="?__elementname"></p>
  </div>
</div>
```

**Comment by idd2000** [comment]
block2__elementname а если нужно изменить стиль для элемента блока, то используй модификаторы
